libs/deps/mas/merch-card.js

```diff
@@ -113,7 +113,7 @@ import{LitElement as Dt}from"../lit-all.min.js";import{LitElement as wt,html as
         right: 0;
         font-size: var(--type-heading-xxs-size);
         font-weight: 500;
-        max-width: 180px;
+        max-width: 30%;
```

Comment:
Sometimes a card can be super-wide spanning over 4 columns. With 30% it will grow more than it is supposed to be. I think the 180px is safer.
Also in some locales 30% might be lower than 180px, causing the labels flowing to next line.

Reply:
@yesil got it. Though fixing the max-width to 180 px has the disadvantage that if the label width becomes 180px in some case and card width is around 300px, then badge can again overlap with the title